modules/custom/openy_system/src/OpenyModulesManager.php

```diff
@@ -19,14 +21,45 @@ class OpenyModulesManager {
    */
   protected $entityTypeManager;
 
+  /**
+   * The contact settings config object.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Configs prefixes that protected from manual deleting.
+   *
+   * This configs will be deleted with entity bundle removing automatically,
+   * so no need to remove them manually.
+   *
+   * @var array
+   * @see OpenyModulesManager->removeEntityBundle().
+   */
+  protected $protectedConfigPrefixes = [
+    'core.entity_form_display',
```

Comment:
let's store this in yml file, rather than hardcoded. Dedicated simple config is fine too

Reply:
@podarok we discussed this with @ddrozdik and decided to leave `protectedConfigPrefixes` inside `OpenyModulesManager`. For simple config we need to support upgrade path in case of any changes. Store this in yml file also not provide any significant advantages and just complicates the logic of these service, so lets keep it simple.
Anyway, I can provide fix if we really need it.

Reply:
I do not think we need a yml file since the list of the items will not be changed frequently.